ui/raidboss/data/02-arr/dungeon/haukke_manor.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+[{
+  zoneId: ZoneId.HaukkeManor,
+  timelineFile: 'haukke_manor.txt',
+  triggers: [
+    {
+      id: 'Haukke Normal Dark Mist DPS/Heals',
+      netRegex: NetRegexes.startsUsing({ id: '2C1', source: 'Manor Claviger || Manor Maidservant || Lady Amandine' }),
```

Comment:
if there are multiple names, you can just list them in an array
```suggestion
      netRegex: NetRegexes.startsUsing({ id: '2C1', source: ['Manor Claviger', 'Manor Maidservant', 'Lady Amandine'] }),
```